platform.md

```diff
@@ -937,7 +937,7 @@ Where:
 - `id`, `version`, and `api` MUST be present for each buildpack
 - `processes` contains the complete set of processes contributed by all buildpacks
 - `slices` contains the complete set of slices defined by all buildpacks
-- `bom` contains the Bill of Materials contributed by buildpacks implementing Buildpack API < 0.7
+- `bom` contains the legacy Bill of Materials contributed by buildpacks (where [supported](buildpack.md))
```

Comment:
I think we should just remove this key from the label in 0.9+, making this change unnecessary.